portal-web/docroot/html/taglib/ui/menu/page.jsp

```diff
@@ -20,12 +20,18 @@
 Menu menu = (Menu)request.getAttribute("liferay-ui:menu:menu");
 
 List<MenuItem> menuItems = menu.getMenuItems();
+
+Optional<MenuItem> itemWithIcon = menuItems.stream(
```

Comment:
As a convention, we need to use the type as part of the variable name. In this case it would be `menuItemOptional`.